packages/actor-rdf-parse-html-script/lib/ActorRdfParseHtmlScript.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  ActorRdfParseFixedMediaTypes,
+  IActionRdfParse,
+  IActionRootRdfParse,
+  IActorOutputRootRdfParse,
+  IActorRdfParseFixedMediaTypesArgs,
+  IActorRdfParseOutput,
+  IActorTestRootRdfParse,
+} from "@comunica/bus-rdf-parse";
+import {ActionContext, Actor, Mediator} from "@comunica/core";
+import { namedNode, quad } from "@rdfjs/data-model";
+import {Readable} from "stream";
+
+/**
+ * A HTML script RDF Parse actor that listens on the 'rdf-parse' bus.
+ *
+ * It is able to extract and parse any RDF serialization from script tags in HTML files
+ * and announce the presence of them by media type.
+ */
+export class ActorRdfParseHtmlScript extends ActorRdfParseFixedMediaTypes {
+
+  public mediatorRdfParse: Mediator<Actor<IActionRootRdfParse, IActorTestRootRdfParse,
+    IActorOutputRootRdfParse>, IActionRootRdfParse, IActorTestRootRdfParse, IActorOutputRootRdfParse>;
+
+  private readonly htmlparser: any;
+
+  constructor(args: IActorRdfParseFixedMediaTypesArgs) {
+    super(args);
+    this.htmlparser = require("htmlparser2");
+  }
+
+  public async runHandle(action: IActionRdfParse, mediaType: string, context: ActionContext):
+    Promise<IActorRdfParseOutput> {
+    const supportedTypes: string[] = Object.keys((await this.mediatorRdfParse
+      .mediate({
+        context,
+        mediaTypes: true,
+      })).mediaTypes);
+    supportedTypes.push("application/ld+json");
```

Comment:
This line should be removed, but this would require fixing #318 first.

Reply:
@brechtvdv FYI, #318 has been fixed now.

Reply:
This works!